TooLongTextAnalyzer.java

```diff
@@ -0,0 +1,9 @@
+public class TooLongTextAnalyzer implements TextAnalyzer {
+
+    private int maxLength = 50;
```

Comment:
change please
private int maxLength;
    public TooLongTextAnalizer(int maxLength) {
        this.maxLength = maxLength;
    }

Reply:
Ok